nautilus_trader/adapters/binance/common/enums.py

```diff
@@ -35,9 +35,21 @@
 from nautilus_trader.model.enums import TimeInForce
 from nautilus_trader.model.enums import TriggerType
 from nautilus_trader.model.enums import bar_aggregation_to_str
+from nautilus_trader.model.identifiers import PositionId
 from nautilus_trader.model.orders import Order
 
 
+@unique
+class BinanceFuturesPositionSide(Enum):
```

Comment:
I'm curious to know why we moved this to common, I had a quick look at the API docs and only saw hedging mode associated with futures?

Reply:
Yes, hedging mode is only associated with futures. Currently all the code for parsing position_id is in common module, and I moved it to the current folder. Maybe it's better to keep it as it is? Feel free to modify the code 🙂